kms/api/Kms.Samples/VerifyAsymmetricEc.cs

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START kms_verify_asymmetric_signature_ec]
+
+public class VerifyAsymmetricSignatureEcSample
+{
+    // Cloud KMS returns signatures in a DER-encoded format. .NET requires
+    // signatures to be in IEEE 1363 format, and converting between these formats
+    // is a few hundred lines of code.
```

Comment:
I guess the idea of doing this in BC is totally out?
I think it would be nice to show it, even if it adds a dependency.  We can always move to System.Security.Cryptography when the functionality is available there.

We can include the `using` statement within the scope of the START/END blocks and add comments to make it clear that a third party library is required.

Reply:
While the refactored samples would make it very clear that we're depending on BC, I'm against encouraging users to take a third-party dependency on software we do not control or audit.